VirtualMachineSetup.md

```diff
@@ -0,0 +1,152 @@
+
+10. Finally, you'll be given a choice of what software to install, with a mix of different desktop environments given. I recommend you check the boxes next to *Debian desktop environment*, *... GNOME*, *SSH server*, and *standard system utilities*.
+
+Note: I have not extensively tested other options here, so if you chose a different desktop environment be aware that you may need to install other packages manually. I did briefly test just choosing Debian desktop environment without GNOME and needed to install a lot of other packages, such as `libjpeg`, `libgif` etc., in order to compile `vxsuite`.
+
+![Software Install Screen](setup_screenshots/software.png?raw=true) 
+
+11. When installation is complete you will be prompted to reboot your system. 
+
+## Install Parallels Tools
+
+Before you can continue with the Debian Quickstart in the main README we will want to install the Parallels tools, this will allow all of the Parallels features, such as copy and pasting between your VM and home computer to work. 
+
+ Log into your new user account, with the user password you set. Click Actions -> Install Parallels Tools, and then click Continue on the pop-up. 
+
+![Install Parallels Tools](setup_screenshots/install_parallels_tools.png?raw=true) 
+
+ Open a terminal (Activities -> Terminal). And enter the command: 
+
+`su - ` Note: the trailing `-` here is important!
+
+Then authenticate with the *root* password. You are now in a root shell. The installation CD will auto-mount in a non-readable manner, to fix this enter the command:
+
+```sh
+mount -t iso9660 -o exec /dev/cdrom /media/cdrom`
+```
+
+If you get an error about it already being mounted, first enter
+
+```sh
+umount /media/cdrom
+```
+
+Then enter the following commands to start the installer: 
+
+```sh
+cd /media/cdrom
+./install
+``` 
+
+Click continue through the prompts and your machine will automatically reboot at the end of this installation process. You can test that it works by copy and pasting things back and forth from the VM and your home machine. 
+
+## Using the Terminal and VS Code on your home machine
+
+If you would be like to develop from your home machines terminal and use VS Code, you will need to do the following things. This assumes a Mac OS.
+
+On your home machine in the terminal open the ssh config file:
+```sh
+vim ~/.ssh/config
+```
+In your VM open the settings application. Click Network, then the gear icon next to the Wired Connected toggle. Copy the IPv4 address.
+In that file add a new entry like so, HOSTNAME will be how you get in and out of your VM, name it anything you would like.
+```
+Host <HOSTNAME> 
+  HostName <HOST_IPv4_ADDRESS>
+  ForwardAgent yes
+```
+If your username on the vm is different then your home computer you will also need to add a line
+```
+  User <VM_USERNAME>
+```
+
+Save the file and test the connection by typing the following command on your home machine
+```
+ssh <HOSTNAME>
+```
+
```

Comment:
```suggestion

If the above command prompted you for a password, run this on your home machine:

\`\`\`
ssh-copy-id <HOSTNAME>
\`\`\`

Then try the `ssh` command again. It should complete without prompting for a password.
```

Reply:
Whoa, I've always just lived with putting my password in all the time lol